tests/system/feature_store/test_feature_store.py

```diff
@@ -205,7 +207,6 @@ def _ingest_quotes_featureset(self):
             quotes_set,
             quotes,
             entity_columns=["ticker"],
-            timestamp_key="time",
```

Comment:
Since we decided to keep the `timestamp_key` for sources, while deprecating it, the tests should contain at least one test (and ideally more) that still use these fields and verify nothing breaks. Can also verify that you're getting the deprecation warning when doing this.
Special attention should be given to the serialization/de-serialization of graphs that use deprecated paths (see my comment on the  `SetEventMetadata` class - should have a test that creates a fset with graph with this step, exporting it, importing it and verifying it still works.

Reply:
We discussed this and agreed to still pass the parameter in at least one test.